tensorflow/lite/examples/python/label_image.py

```diff
@@ -54,10 +54,40 @@ def load_labels(filename):
       help='input standard deviation')
   parser.add_argument(
       '--num_threads', default=None, type=int, help='number of threads')
+  parser.add_argument(
+      '-e',
+      '--ext_delegate',
+      help='external_delegate_library path')
+  parser.add_argument(
+      '-o',
+      '--ext_delegate_options',
+      help='external delegate options, \
+            format: "option1: value1; option2: value2"')
+
   args = parser.parse_args()
 
+  ext_delegate = None
+  ext_delegate_options = {}
+
+  # parse extenal delegate options
+  if args.ext_delegate_options is not None:
+    options = args.ext_delegate_options.split(';')
+    for o in options:
+      kv = o.split(':')
+      if(len(kv) == 2):
```

Comment:
Raise an error if there is a wrongly formatted option?

Reply:
Done.